### ISSUE TITLE
Missing Axe Core Rules

### ISSUE THREAD

**Issue by emilyrohrbough** [cerner/terra-toolkit#299]
# Bug Report

## Description
Axe core requires each compliance set to be added. We are missing `wcag2a` and `wcag21aa`.

## Mentions
@bjankord
